R/tm_variable_browser.R

```diff
@@ -432,7 +427,6 @@ srv_variable_browser <- function(id, datasets, reporter, datasets_selected, ggpl
         card <- teal.reporter::TealReportCard$new()
         card$set_name("Variable Browser Plot")
         card$append_text("Variable Browser Plot", "header2")
-        card$append_text("Filter State", "header3")
```

Comment:
Hmm so we need to record whether the use filtered data button was turned on or not as otherwise we could have the situation where say "Filtered State: x" but the Use filtered state button is set to FALSE and then the graph includes unfiltered data. Maybe we have `if (input$<<use_filter>>) card$append_fs(...)`

Reply:
good point, updated.